settings.php

```diff
@@ -14,7 +14,7 @@
 
 // Handling of PHP internal errors
 $last_error = error_get_last();
-if($last_error["type"] === E_WARNING || $last_error["type"] === E_ERROR)
+if($last_error && ($last_error["type"] === E_WARNING || $last_error["type"] === E_ERROR))
```

Comment:
I'm not sure if checking a variable for truthy/falsey is good or not but I think a safer approach would be to check the length of `$last_error`s associative array for `>0` instead.

Reply:
I just thought since `error_get_last()` either outputs an array or NULL if there was no error, it would be simpler to just check if it's true

Reply:
Okay, as long as Javascript will always consider `NULL` to be falsy then that's okay. A nested `if` might look cleaner, but that's more a style issue. 

I left these as comments and not change requests on purpose, this shouldn't block approval and merging.

Reply:
```suggestion
if(isset($last_error) {
 if($last_error["type"] === E_WARNING || $last_error["type"] === E_ERROR)) {
   do stuff
  }
}
```

Reply:
> Is something like `isset($last_error)` an option?

No, because the line
```php
$last_error = error_get_last();
```

will always set the variable. It may have been set to `NULL`, so an explicit check would look like
```suggestion
if(!is_null($last_error) && ($last_error["type"] === E_WARNING || $last_error["type"] === E_ERROR))
```

Reply:
>isset() will return FALSE when checking a variable that has been assigned to NULL. Also note that a null character ("\0") is not equivalent to the PHP NULL constant.

Reply:
Oh, sorry. I have overlooked this bit, it seems.

Reply:
Personally happy with non-nested - but agree with the suggestion for `isset($last_error)`, as it's more explicit (though potentially does the same thing!

Reply:
So we make this a

```suggestion
if(isset($last_error) && ($last_error["type"] === E_WARNING || $last_error["type"] === E_ERROR))
```

Reply:
Feel free to commit the suggestion if the OP doesn't respond.

Reply:
Will just do it anyway. ;)